xbmc/music/MusicDatabase.cpp

```diff
@@ -2945,11 +2945,11 @@ void CMusicDatabase::Clean()
   // other writing access to the database is prohibited.
   if (g_application.IsMusicScanning())
   {
-    CGUIDialogOK::ShowAndGetInput(189, 14057, 0, 0);
+    CGUIDialogOK::ShowAndGetInput(189, 14057);
     return;
   }
 
-  if (CGUIDialogYesNo::ShowAndGetInput(313, 333, 0, 0))
+  if (CGUIDialogYesNo::ShowAndGetInput(313, 333))
```

Comment:
well, these changes should be applied to all the yes/on and progress dialogs as well :-)